### ISSUE TITLE
Clarify Udacity assignment #2

### ISSUE THREAD

**Issue by vincentvanhoucke** [tensorflow/tensorflow#1284]
- expand on 'hiddent'.
- Point to nn.relu:
https://www.tensorflow.org/versions/r0.7/api_docs/python/nn.html#relu

**Comment by aahoo** [comment]
@vincentvanhoucke could you please clarify this assignment a little bit more. 

Problem: "Turn the logistic regression example with SGD into a 1-hidden layer neural network with rectified linear units (nn.relu()) and 1024 hidden nodes. This model should improve your validation / test accuracy."

I modified the code as below and the accuracy improved by about 2 percent. But I don't think it is correct. 

```
  # Training computation.
  logits = tf.matmul(tf_train_dataset, weights) + biases
  
  # modification is the line below
  relu_output = tf.nn.relu(logits)

  loss = tf.reduce_mean(
    tf.nn.softmax_cross_entropy_with_logits(relu_output, tf_train_labels))
  
  # Optimizer.
  optimizer = tf.train.GradientDescentOptimizer(0.5).minimize(loss)
```
I did not use 1024 hidden nodes. In fact I don't know what it means. Is it the batch size?

How much approximately should the accuracy improve?
